src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1269,9 +1269,11 @@ CHIP_ERROR CASESession::ConstructTBS3Data(const uint8_t * responderOpCert, uint3
 
 CHIP_ERROR CASESession::ComputeIPK(const uint16_t sessionID, uint8_t * ipk, size_t ipkLen)
 {
+    uint8_t sid[2] = { (uint8_t) sessionID, (uint8_t)(sessionID >> 8) };
```

Comment:
use byte endianess macros please. LittleEndian::BufferWriter seems appropriate

Reply:
This code will go away in @jpk233 's PR since it's already gone there, due to not being spec. I think this is OK for a hotfix in TE4 for dead code.

Reply:
Using buffewriter is a trivial change. CI takes 3-4 hours anyway. Please do the change.

Reply:
Done.